docs/apv.md

```diff
@@ -324,9 +324,23 @@ De overtreding beschreven in lid 1 staat bekend als “spam”
 4. Het is niet toegestaan om jezelf voor te doen als overheidsmedewerker bij het ontsnappen of helpen ontsnappen van een derde persoon uit het cellencomplex.
 5. Bij overtreding van lid 1, lid 2, lid 3 en/of lid 4 zal er een straf uitgedeeld worden volgens de 1e categorie en zal het roleplayscenario worden teruggedraaid.
 
+### Artikel 38 - Witwassen
+
+1. Witwassers overvallen is toegestaan wanneer minimaal één van ondergenoemde punten van toepassing is.
+    * Hij/zij beschermende kleding zoals een body armor draagt.
+    * Gezichtsbedekking op heeft tijdens het wassen.
+    * Een overduidelijke escort heeft.
+2. Witwassen mag enkel in de voertuigen die je krijgt van het depot.
+3. Een persoon herkennen uit het criminele circuit valt onder powergaming en is geen reden om diegene te overvallen.
+4. Afleverpunten herkennen valt onder powergaming.
```

Comment:
Weet niet of deze er in moet blijven aangezien er dan geen risico meer aan zit. herkennen nee maar het moet niet zo zijn dat je niet meer overvallen kan worden daar lijkt me